ballerina/client.bal

```diff
@@ -66,7 +66,10 @@ public distinct isolated client class Client {
             params: {
                 protocolVersion: LATEST_PROTOCOL_VERSION,
                 capabilities: config.capabilityConfig?.capabilities ?: {},
-                clientInfo: config.info
+                clientInfo: config.info ?: {
+                    name: "MCP Client",
+                    version: "1.0.0"
+                }
```

Comment:
Cant we specify this only for major versions

Reply:
Something like "1"

Reply:
Isn't this the SemVer format?

Reply:
Instead of allowing nil, Cant we make info as default parameter?